projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -51,12 +52,32 @@ public function __construct() {
 			add_action( 'admin_post_jetpack_revoke_invite_user_to_wpcom', array( $this, 'handle_request_revoke_invite' ) );
 			add_action( 'admin_post_jetpack_resend_invite_user_to_wpcom', array( $this, 'handle_request_resend_invite' ) );
 			add_action( 'admin_print_styles-users.php', array( $this, 'jetpack_user_table_styles' ) );
-			add_action( 'admin_print_styles-user-new.php', array( $this, 'jetpack_user_new_form_styles' ) );
 			add_filter( 'users_list_table_query_args', array( $this, 'set_user_query' ), 100, 1 );
+			add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
 
 			self::$tracking = new Tracking();
 		}
 
+		/**
+		 * Enqueue assets for user-new.php.
+		 *
+		 * @param string $hook The current admin page.
+		 */
+		public function admin_enqueue_scripts( $hook ) {
+			if ( 'user-new.php' === $hook ) {
```

Comment:
This is a fine way to do it, especially if you anticipate extending it to other admin pages.

If you wanted to avoid this callback to run on every wp-admin request, you could also use the [`'admin_print_scripts-user-new.php'`](https://github.com/WordPress/wordpress-develop/blob/6.4/src/wp-admin/admin-header.php#L139) action.